cvat-ui/src/components/cvat-app.tsx

```diff
@@ -39,6 +39,7 @@ import showPlatformNotification, {
     showUnsupportedNotification,
 } from 'utils/platform-checker';
 import '../styles.scss';
+import EmailConfirmationMessage from './email-confirmation-page/email-confirmed';
```

Comment:
Would rename it to ``EmailConfirmationPage``, what do you think?

Reply:
yes sure, 
done